network/tc.go

```diff
@@ -19,21 +19,20 @@ import (
 
 const tcPath = "/sbin/tc"
 
-// default protocol identifiers from /etc/protocols
+type ConnState string
```

Comment:
🌷 
```suggestion
type connState string
```

If you keep this type private, you guarantee nobody can create a new value of this type outside of this package.
It's one way to ensure you can trust this type values to be always valid.

Reply:
Indeed this would require a small change that is IMO ok (this keeps all ConnState decision to the same package), here is what I had in mind:
* https://github.com/DataDog/chaos-controller/compare/network_disruption_u32_flower...network_disruption_u32_flower-ex-prv
I still may have missed some usage 🙈

Reply:
LGTM, I'll merge that in there